staging/src/k8s.io/apiserver/pkg/admission/configuration/manager.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Please don't name this file manager.go. It's much too generic, and if we ever use glog in here, it'll be difficult to know which "manager.go" it's for. I'd recommend configuration_manager.go or something like that.